projects/core/src/checkout/store/effects/payment-types.effect.ts

```diff
@@ -46,7 +49,15 @@ export class PaymentTypesEffects {
           payload.poNumber
         )
         .pipe(
-          map((data) => new CheckoutActions.SetPaymentTypeSuccess(data)),
+          mergeMap((data) => {
```

Comment:
Perhaps a switchMap or concatMap? We probably don't want multiple payment types updates happening in parallel.

Reply:
`switchMap` is used in line `43` to prevent multiple payment types updates happening in parallel. But, if we get the response, we want to process it (invoke some actions), so here is `mergeMap`.